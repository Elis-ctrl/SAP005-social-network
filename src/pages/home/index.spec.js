import { Home } from './index.js';
// import {currentUser} from '../../services/index.js';

describe('Home', () => {
  it('should be a function', () => {
    expect(typeof Home).toBe('function');
  });
  it('shoul return homepag', () => {
    expect(Home()).toMatchSnapshot();
  });
});
