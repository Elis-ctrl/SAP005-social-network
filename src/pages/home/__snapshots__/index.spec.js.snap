// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home shoul return homepag 1`] = `
<div
  class="div-home"
>
  
  
  <div
    id="fundo"
  />
  
      
  <figure
    class="logo"
  >
    
      
    <img
      alt="Logo Runners"
      id="logo"
      src="./assets/logo_runners.png"
    />
    
      
  </figure>
  
      
  <div
    class="home-box"
  >
    
        
    <form
      class="login"
    >
      
          
      <div
        class="input-form"
      >
        
            
        <p
          alt="Login"
          class="login"
        >
          Login
        </p>
        
            
        <h3
          class="error"
          id="msgError"
        />
        
            
        <input
          id="email-input"
          placeholder="@E-mail"
          required=""
          type="email"
        />
        
            
        <input
          id="password-home"
          placeholder="Senha"
          required=""
          type="password"
        />
        
            
        <button
          id="submit-home"
        >
          Entrar
        </button>
        
            
        <p
          class="text"
        >
          Logar com:
        </p>
        
            
        <div
          class="button-social"
          id="authGoogle"
        >
          
            
          <input
            alt="Logo Google"
            class="google"
            id="authGoogle"
            type="button"
          />
          
            
        </div>
        
          
      </div>
      
        
    </form>
    
      
  </div>
  
      
  <div
    class="register-home"
  >
    
      
    <p
      class="sub-text"
    >
      NÃ£o tem uma conta? 
      <a
        href="/register"
      >
        Cadastre-se
      </a>
    </p>
    
      
  </div>
  
  
</div>
`;
